Purpose.md

```diff
@@ -0,0 +1,23 @@
+# The Wardley Mapping Foundation's Purpose
+The foundation exists to promote, sustain and increase knowledge of the mapping of competitive landscapes for individuals and collectives around the world, for the public benefit.
+
+To achieve this, its focus is to:
+1. Create, maintain, share and support open repositories of knowledge that are free to use
+2. Encourage and facilitate research to advance public knowledge
+3. Facilitate the creation of open training materials and public education programmes
+4. Facilitate the hosting of events focused on educating the public and sharing knowledge
+5. Encourage and facilitate the creation of open standards and tooling that improve the state of mapping
+6. Support the growth of an international community of people and organisations that share our values
+
+# Operating Principles
+We use Simon's 40 guides to help us make better decisions.
+![40 guides to better decisions](https://pbs.twimg.com/media/ErsAqc4WMAAz5jL?format=jpg&name=large)
+
+# Values
+Shalom Schwartz' [Theory of Basic Values](https://doi.org/10.9707/2307-0919.1116), and Oliver Scott Curry's [7 Universal Moral Rules](https://doi.org/10.1086/701478) both suggest that people share a universal set of human values regardless of race, ethniticy, gender, etc.
+
+We subscribe to Curry's premise that **cooperative interactions are non-zero-sum games**.  We believe that by understanding competitive landscapes, individuals & collectives can influence the mode of competition itself.
+
```

Comment:
As a somewhat casual mapper, I was unaware of this theory.

Have downloaded and will read, but although I can say I fully subscribe to the values of Open and Inclusive, I cannot say that I understand the theory behind them and it would take me a long while to say that I did so truthfully.